### ISSUE TITLE
Split-user: wconfd cannot access client.pem

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1008]
Originally reported of Google Code with ID 952.

```
In the split-user scenario, client.pem is owned by gnt-masterd:gnt-masterd, with permissions of 440. gnt-wconfd is not a member of the gnt-masterd group.

I'm not sure what's wrong here, the ownership of client.pem or the group memberships of the wconfd user.
```
Originally added on 2014-09-23 07:48:37 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 9decc439eb4a996bc5b7ac8a6852be545a494ecd
Author: Apollon Oikonomopoulos <apoikos@gmail.com>
Date:   Tue Sep 23 15:09:05 2014 +0300

    WConfd: run as <prefix>-masterd by default
    
    Since WConfd is taking over the old masterd role, it is best to have it
    run as the old masterd user. A lot of the files (notably config.data and
    client.pem) are accessible by masterd-only.
    
    This is really just to ease the transition from masterd to wconfd and
    get things working again and should be re-evaluated in the long run.
    
    This fixes issues #952 and #953.
    
    Signed-off-by: Apollon Oikonomopoulos <apoikos@gmail.com>
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
 
```
Originally added on 2014-09-24 14:47:05 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-10-10 11:40:54 +0000 UTC.

Changed State: Released
